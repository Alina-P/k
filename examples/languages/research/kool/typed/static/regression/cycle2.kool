class main extends main {
  method main() {
    print("No cycle detected\n");
  }
}

class main extends main {
  method main() {
    print("No cycle detected\n");
  }
}

