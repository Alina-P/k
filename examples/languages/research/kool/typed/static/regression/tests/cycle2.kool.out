Class "main" is in a cycle!
